glimmer-version/app/templates/components/focusing-outlet.hbs

```diff
@@ -0,0 +1,3 @@
+{{#focusing-inner outletState=(-get-dynamic-var "outletState") focus=(action "checkFocus")}}
```

Comment:
`glimmer-version` isn't super good naming wise, since technically both versions have been *retconned* into being "glimmer versions" (since 1.13 was called Glimmer v1).  I'd suggest `current` and `legacy-lt-2-10` or something like that.